commcare_connect/opportunity/models.py

```diff
@@ -243,9 +245,7 @@ def learn_progress(self):
         learn_modules_count = learn_modules.count()
         if learn_modules_count <= 0:
             return 0
-        completed_modules = CompletedModule.objects.filter(
-            opportunity=self.opportunity, module__in=learn_modules, user=self.user
-        ).count()
+        completed_modules = self.completedmodule_set.count()
```

Comment:
same with this? is this equivalent behavior?

Reply:
Yup, the filter for `module_in=learn_modules` is extra as the learn_modules could not be removed once the Opportunity is created so directly fetching completed_modules related to the opportunity_access for a user also gives the same output.